### ISSUE TITLE
Contact Me here

### ISSUE THREAD

**Issue by smartnavdeep** [smartnavdeep/smartnavdeep#2]
# Contact Me here 😄
You can contact me here if you want to by commenting in this issue

**Comment by Ved-programmer** [comment]
great

**Comment by smartnavdeep** [comment]
😀

**Comment by Ved-programmer** [comment]
By the way, your log in system is a few commits behind mine. So you might want to update that

**Comment by Ved-programmer** [comment]
hi, someone stole our code, I have mailed you the details. But still, if you are not able to check the mail then here it is:

https://code-projects.org/numerical-base-converter-in-python-with-source-code/
https://www.youtube.com/watch?v=ze6SZO8M7-w

**Comment by smartnavdeep** [comment]
OH my god ! Well in github we can report the user to github, but for that the user must be in github i.e He must have a github account.
Btw we did add a licence so it will be easy to report github about his behavior

**Comment by Ved-programmer** [comment]
true, I also reported the youtube video. I don't think Github would be able to do anything much since we don't know about this person's Github account. But we can contact youtube since the video is there. Let's wait for youtube to answer and then see what happens. I will keep you updated.

**Comment by smartnavdeep** [comment]
I just sent an email to you.

**Comment by Ved-programmer** [comment]
ok I will check that
